public/jsx/views/card-details.jsx

```diff
@@ -57,7 +57,7 @@ module.exports = React.createClass({
         </div>
       );
     } else {
-      return <Spinner text={gettext('Setting up')}/>;
```

Comment:
Is there any difference to doing the gettext in the class as opposed to in the JSX? It looks like more code, but the result is same.